app/views/case_workers/admin/management_information/_stats_report.html.haml

```diff
@@ -0,0 +1,17 @@
+- i18n_scope = 'case_workers.admin.management_information.index'
+
+- if report_rec&.started_at
+  = t('report_generated_at_html',
+      scope: i18n_scope,
+      datetime: report_rec.started_at.iso8601,
+      time: report_rec.started_at.strftime(Settings.report_date_format))
+- else
+  = t('unavailable_report', scope: i18n_scope)
+
+%p
```

Comment:
empty `<p>` tag

Reply:
should probably be a <br≥? There were spacing issues without this i believe

Reply:
fix here 8016efdcf